conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosGovernanceOperations.scala

```diff
@@ -83,17 +83,25 @@ object TezosGovernanceOperations extends LazyLogging {
     }
 
     // as stated, this is a runtime failure if the block contains the wrong metadata
-    def proposalOperationsHashes(block: Block) = block.data.metadata match {
+    def proposalHashesPerCycle(block: Block) = block.data.metadata match {
       case BlockHeaderMetadata(_, _, _, _, _, level) =>
         TezosDatabaseOperations.getProposalOperationHashesByCycle(level.cycle)
     }
 
     logger.info("Searching for governance data in voting period...")
 
+    //find blocks for a specific proposal under scrutiny, relevant for counting ballots
+    val votingBlocks = proposalsBlocks.collect { case (block, Some(protocol)) => block }.toList
+
+    logger.info(
+      "There are {} blocks related to some voting - excluding proposals.",
```

Comment:
Let's make this a bit clearer: "There are {} blocks related to testing vote and proposal vote periods..".

Reply:
done, thanks